lib/libc/net/sctp_send.3

```diff
@@ -48,11 +49,25 @@
 .Fa "int sd" "const void *msg" "size_t len" "struct sockaddr *addrs"
 .Fa "int addrcnt" "const struct sctp_sndrcvinfo *sinfo" "int flags"
 .Fc
+.Ft ssize_t
+.Fo sctp_sendv
+.Fa "int sd" "const struct iovec *iov" "int iocnt" "struct sockaddr *addrs"
+.Fa "int addrcnt" "void *info" "socklen_t infolen" "unsigned int infotype"
+.Fa "int flags"
+.Fc
 .Sh DESCRIPTION
 The
 .Fn sctp_send
 system call
 is used to transmit a message to another SCTP endpoint.
+The
+.Fn sctp_sendx
+function is used to transmit a message to multiple endpoint addresses.
```

Comment:
Maybe something like
When issuing an implicit association setup, the
.Fn sctp_sendx
function allows to specify multiple addresses of the peer, which can then be used for the association setup.
This is equivalent to using
.Xr sctp_connectx 3 .

Reply:
i replaced the text with this, which i think is still accurate, but is slightly shorter to fit the style of the paragraph:

```
The
.Fn sctp_sendx
function is used to specify multiple peer addresses when creating an implicit
association, as in
.Xr sctp_connectx 3 .
```